docs/modules/ROOT/pages/autotasks.adoc

```diff
@@ -55,12 +55,38 @@ const { Relayer } = require('defender-relay-client');
  
 exports.handler = async function(event) {
   const relayer = new Relayer(event);
-  // Use relayer for sending txs or querying the network
+  // Use relayer for sending txs or querying the network...
 }
 ----
 
 This allows you to send transactions using your Relayer from your Autotasks without having to set up any API keys or secrets. Furthermore, you can also use the Relayer JSON RPC endpoint for making queries to any Ethereum network without having to configure API keys for external network providers.
 
+If you want to https://www.npmjs.com/package/defender-relay-client#ethersjs[use `ethers.js`] for making queries or sending transactions via your Relayer, change the above to:
+
+[source,jsx]
+----
+const { DefenderRelaySigner, DefenderRelayProvider } = require('defender-relay-client/lib/ethers');
+const ethers = require('ethers');
+ 
+exports.handler = async function(event) {
+  const provider = new DefenderRelayProvider(event);
+  const signer = new DefenderRelaySigner(event, provider, { speed: 'fast' });
```

Comment:
```suggestion
  const signer = new DefenderRelaySigner(event, provider, { speed: 'fast' });
  // Now you can use the `provider` or `signer` as you'd usually do with ethers
  // ...
```
I'd add something like this, or even a more specific usage example if you feel like, just to show that the `provider` and `signer` can be used with ethers. Otherwise is like we're importing `ethers` in the example but we're not using it at all in the code, which could be confusing for more beginners.